proto/steward/v4/cellar_v2.proto

```diff
@@ -196,3 +191,290 @@ message CellarV2Governance {
+        // Represents function `liftShutdown()`
+        LiftShutdown lift_shutdown = 7;
+        // Represents function `forcePositionOut(uint32 index, uint32 positionId, bool inDebtArray)`
+        ForcePositionOut force_position_out = 8;
+        // Represents function `toggleIgnorePause(bool ignore)`
+        ToggleIgnorePause toggle_ignore_pause = 9;
+    }
+
+    /*
+    * Allows the owner to add an adaptor to the Cellar's adaptor catalogue
+    *
+    * Represents function `addAdaptorToCatalogue(address adaptor)`
+    */
+    message AddAdaptorToCatalogue {
+        string adaptor = 1;
+    }
+
+    /*
+    * Allows the owner to add a position to the Cellar's position catalogue
+    *
+    * Represents function `addPositionToCatalogue(uint32 positionId)`
+    */
+    message AddPositionToCatalogue {
+        uint32 position_id = 1;
+    }
+
+    /*
+    * Allows share lock period to be updated.
+    *
+    * Represents function `setShareLockPeriod()`
+    */
+    message SetShareLockPeriod {
+        string new_lock = 1;
+    }
+
+    /*
+    * Changes the cellar's allowed rebalance deviation, which is the percent the total assets of a cellar may deviate
+    * during a `callOnAdaptor`(rebalance) call. The maximum allowed deviation is 100000000000000000 (0.1e18), or 10%.
+    *
+    * Represents function `setRebalanceDeviation(uint256)`
+    */
+    message SetRebalanceDeviation {
+        string new_deviation = 1;
+    }
+
+    /*
+    * Shutdown the cellar. Used in an emergency or if the cellar has been deprecated.
+    *
+    * Represents function `initiateShutdown()`
+    */
+    message InitiateShutdown {}
+
+    /*
+    * Allows strategist to set the platform cut for the cellar.
+    *
+    * Represents function `setStrategistPlatformCut(uint64 cut)`
+    */
+    message SetStrategistPlatformCut {
+        // The new strategist platform cut
+        uint64 new_cut = 1;
+    }
+
+    /*
+    * Allows the owner to restart a shut down Cellar
+    *
+    * Represents function `liftShutdown()`
+    */
+    message LiftShutdown {}
+
+    /*
+    * Allows caller to force a position out of the cellar
+    *
+    * Represents function `forcePositionOut(uint32 index, uint32 positionId, bool inDebtArray)`
+    */
+    message ForcePositionOut {
+        uint32 index = 1;
+        uint32 position_id = 2;
+        bool in_debt_array = 3;
+    }
+
+    /*
+    * Allows caller to toggle the ignorePause flag on the cellar
+    *
+    * Represents function `toggleIgnorePause(bool ignore)`
+    */
+    message ToggleIgnorePause {
+        bool ignore = 1;
+    }
+}
+
+// Represents a call to adaptor an. The cellar must be authorized to call the target adaptor.
+message AdaptorCall {
+    // Address of the adaptor
+    string adaptor = 1;
+    // The function call data for the adaptor
+    oneof call_data {
+        // Represents function calls to the AaveATokenAdaptor V1
+        AaveATokenAdaptorV1Calls aave_a_token_v1_calls = 3;
```

Comment:
Are we also skipping from id 1 to 3 on 3.x-main?

Reply:
Ah this was because I removed the original bugged uniswap adaptor

Reply:
We can modify ID values with a major proto version update.